Formula/fheroes2.rb

```diff
@@ -0,0 +1,39 @@
+class Fheroes2 < Formula
+  desc "Free Heroes of Might and Magic II () is a recreation of HoMM2 game engine"
+  homepage "https://ihhub.github.io/fheroes2/"
+  url "https://github.com/ihhub/fheroes2/archive/refs/tags/0.9.7.tar.gz"
+  sha256 "02e777485906b8112636173c7e241d2824b2dce7707b5ebf6b3bab70ea9865f7"
+  license "GPL-2.0-or-later"
+  head "https://github.com/ihhub/fheroes2.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :git
+    regex(/^((\d+\.)+(\d)+)$/i)
+  end
```

Comment:
```suggestion
```
We really don't need this. This is just a less good version of the default `livecheck`.

Reply:
Fair enough, but let's try to use a standard regex:
```suggestion

  livecheck do
    url :stable
    strategy :git
    regex(/^v?(\d+(?:\.\d+)+)$/i)
  end
```